src/recordlinker/schemas/pii.py

```diff
@@ -357,31 +372,40 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
             for address in self.address:
                 if address.postal_code:
                     # only use the first 5 digits for comparison
-                    yield address.postal_code[:5]
+                    yield normalize_text(address.postal_code)[:5]
```

Comment:
If we were to make removing of leading/trailing whitespace a normal part of the PIIRecord field validation process, we wouldn't need to call normalize_text here right?

Reply:
You're right, we wouldn't need to use `normalize_text` to remove the whitespace here.

Reply:
To elaborate more on that, I was wondering if it be good to implement something like the following.  Then have all our pii.py classes inherit from it.

```python
class StrippedBaseModel(BaseModel):
    @field_validator('*', mode='before')
    def strip_whitespace(cls, v):
        if isinstance(v, str):
            return v.strip()
        return v
```

Reply:
Yep, in agreement!